packages/jaeger-ui/src/model/trace-dag/DagNode.js

```diff
@@ -36,7 +37,12 @@ export default class DagNode<T = void> {
     this.parentID = parentID;
     this.id = DagNode.getID(service, operation, hasChildren, parentID);
     this.count = 0;
+    this.members = [];
```

Comment:
Per @everett980's comment, `.count` can be turned into a getter to avoid having to maintain it:

```js
set count(value) {
  throw new Error('Illegal operation, use .addMember(...)');
}
get count() {
  return this.members.length;
}
```

Reply:
Great point. We can leave dealing with that for another day.